docs/gettingstarted/quickstart.rst

```diff
@@ -0,0 +1,221 @@
+:layout: homepage
+
+============================
+Getting started with Flocker
+============================
+
+.. logo::
+
+.. intro-text::
+
+   Flocker lets you run microservices apps with database containers and move them around between servers. It comes in two pieces and you’ll need both.
+
+.. contents::
+   :local:
+
+.. tutorial-step::
+
+   Step 1: Installing Flocker CLI & Node
+   =====================================
+
+   .. parallel::
+
+      .. mobile-label::
+
+         Local
+
+      .. image:: images/macbook.png
+         :class: center-block img-responsive
+         :alt: Flocker CLI diagram
+
+      Flocker CLI
+      -----------
+
+      +--------------------------------------------------------------------------+
+      | Runs on your laptop                                                      |
+      +--------------------------------------------------------------------------+
+      | Uses application and deployment configuration files                      |
+      +--------------------------------------------------------------------------+
+      | Deploys containers to a cluster of servers                               |
+      +--------------------------------------------------------------------------+
+
+      .. empty-div:: arrow-down center-block hidden-xs hidden-sm
+
+   .. parallel::
+
+      .. mobile-label::
+
+         Live
+
+      .. image:: images/nodes.png
+         :class: center-block offset-top img-responsive
+         :alt: Flocker Node diagram
+
+      Flocker Node
+      ------------
+
+      +--------------------------------------------------------------------------+
+      | Runs on each server in a cluster                                         |
+      +--------------------------------------------------------------------------+
+      | Links, ports and volumes work across hosts                               |
+      +--------------------------------------------------------------------------+
+      | After deployment, containers can move around                             |
+      +--------------------------------------------------------------------------+
+
+      .. empty-div:: arrow-down center-block
+
+   .. parallel::
+
+      .. mobile-label::
+
+         Local
+
+
+      Installation
+      ------------
+
+      .. tabs::
+
+         OS X
+         ^^^^
+
+         Install the flocker-cli client on your Mac (requires Homebrew):
+
+         .. code-block:: console
+
+            $ brew update && \
+              brew tap clusterhq/flocker && \
+              brew install flocker-0.3.2
+
+         Linux
+         ^^^^^
+
+         Install the flocker-cli client on your Linux machine:
+
+         .. code-block:: console
+
+            $ virtualenv flocker-tutorial && \
+              flocker-tutorial/bin/pip install --upgrade pip && \
+              flocker-tutorial/bin/pip install --quiet flocker-cli
```

Comment:
This step fails because there is no `flocker-cli` package on PyPI (nor have we any plans for one, as far as I know :/).